docs/source/02_get_started/04_configuration.md

```diff
@@ -391,6 +420,24 @@ default_images:
   jupyterlab: "quansight/qhub-jupyterlab:b89526c59a5c269c776b535b887bd110771ad601"
   dask_worker: "quansight/qhub-dask-worker:b89526c59a5c269c776b535b887bd110771ad601"
 
+theme:
+  jupyterhub:
+    hub_title: QHub - thisisatest
+    hub_subtitle: Autoscaling Compute Environment
+    welcome: |
+      Welcome to jupyter.github-actions.qhub.dev. It is maintained by <a href="http://quansight.com">Quansight
+      staff</a>. The hub's configuration is stored in a github repository based on
+      <a href="https://github.com/Quansight/qhub/">https://github.com/Quansight/qhub/</a>.
```

Comment:
The repository needs to change to `qhub-cloud`, but that fine, we can change it later.